apps/server/src/infra/schulconnex-client/response/sanis-erreichbarkeiten-response.ts

```diff
@@ -0,0 +1,11 @@
+import { IsOptional, IsString } from 'class-validator';
+
+export class SanisErreichbarkeitenResponse {
+	@IsOptional()
+	@IsString()
+	typ?: string;
```

Comment:
According to the schulconnex v1.4 documentation as long as you have a "erreichbarkeiten" then "typ" and "kennung" should be defined. Also the object should only be valid from our point of view if both are defined, so they should not be optional.